bower.json

```diff
@@ -23,5 +23,11 @@
     "bower_components",
     "test",
     "tests"
-  ]
+  ],
```

Comment:
While this change is correct, it doesn't belong to this PR as it addresses a different issue.